src/status_im/ui/screens/accounts/recover/styles.cljs

```diff
@@ -31,6 +33,150 @@
    :font-size      12
    :color          colors/gray})
 
-(def recovery-phrase-input
-  {:flex                1
-   :text-align-vertical :top})
+(defn recovery-phrase-input [processing?]
+  {:position :absolute
+   :left "6.4%"
```

Comment:
as you may see in other styles we don't use percents, better to use margin in px

Reply:
@flexsurfer Got it, have been trying to use the markup from Figma as verbatim as possible, the Figma markup is in percent.

Reply:
@flexsurfer Just double-checking this, if the designer has used percents in the design, isn't it better to stick as closely as possible to the designer's intention?